src/Security/Authentication/Negotiate/test/Negotiate.Test/NegotiateHandlerTests.cs

```diff
@@ -208,7 +208,9 @@ public async Task AuthHeaderAfterNtlmCompleted_ReAuthenticates(bool persist)
     public async Task RBACClaimsRetrievedFromCacheAfterKerberosCompleted()
     {
         var claimsCache = new MemoryCache(new MemoryCacheOptions());
-        claimsCache.Set("name", new string[] { "CN=Domain Admins,CN=Users,DC=domain,DC=net" });
+        var claimsList = new List<KeyValuePair<string, string>>();
+        claimsList.Add(new KeyValuePair<string, string>("http://schemas.microsoft.com/ws/2008/06/identity/claims/role", "CN=Domain Admins,CN=Users,DC=domain,DC=net"));
```

Comment:
Can we test caching the `Sid` and `GroupSid` claims now?

Reply:
Well this does not actually test much. It just adds the claims to the cache which is then directly copied to the claims (regardless of the claim types).